.github/workflows/docs-links.yml

```
@@ -0,0 +1,59 @@
+# This lane checks the *external* URLs with lychee.
+#
+# Advisory by design:
+#   - On pull requests it never fails the check (`fail: false`) — a merge must
+#     not be blocked because someone else's docs host had a bad minute. It is
+#     also intentionally NOT a required status check.
+#   - On the weekly schedule it DOES fail (`fail: true`) so link rot that
+#     appears without us touching a file surfaces as a failed run we get
+#     notified about — which is exactly how the Plus-FAT 404 slipped in.
+name: Docs Links
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)